content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Migrate from PodSecurityPolicy
+reviewers:
+- tallclair
+content_type: task
```

Comment:
In the front matter, also set `min-kubernetes-server-version`, and mention the feature gate in the Prerequisites section.